pkg/signup/service/signup_service_test.go

```diff
@@ -956,7 +956,7 @@ func (s *TestSignupServiceSuite) TestGetSignupBannedUserEmail() {
 	require.ErrorAs(s.T(), err, &e)
 	require.Nil(s.T(), response)
 	require.Equal(s.T(), "Failure", e.ErrStatus.Status)
-	require.Equal(s.T(), "forbidden: Access to the Developer Sandbox has been suspended due to suspicious activity or detected abuse.", e.ErrStatus.Message)
+	require.Equal(s.T(), "forbidden: "+service.BannedUserMessage, e.ErrStatus.Message)
```

Comment:
Might also be good to test:
```go
require.True(t, errors.IsForbidden(e))
```

Reply:
Ah, I somehow overlooked the line below.

That said, why do you go to all that length to basically do the equivalent of `require.True(t, errors.IsForbidden(e))`?

Reply:
It was a blind copy-paste - I used the pre-created error as suggested below, and use that also for comparison ac0a1b7